### PR TITLE
AWS Lambda SDK: Send captured events to dev mode

### PR DESCRIPTION
**Pull request by medikoo**

### CODE REVIEW

File: node/packages/aws-lambda-sdk/instrument/lib/auto-send-spans.js [hunk 1]
```diff
@@ -15,37 +15,47 @@ const sendTelemetry = require('./send-telemetry');
...
 const pendingSpans = [];
+const pendingCapturedEvents = [];
 let isScheduled = false;
 let timeoutId = null;
-const sendSpans = () => {
+const sendData = () => {
   isScheduled = false;
   clearTimeout(timeoutId);
-  if (!pendingSpans.length) return;
+  if (!pendingSpans.length && !pendingCapturedEvents.length) return;
```
Comment (Danwakeem) [inline node/packages/aws-lambda-sdk/instrument/lib/auto-send-spans.js:24]: Is it possible to have 0 pending spans and 1 or more pending captured events? 🤔

**Review by Danwakeem — APPROVED** [review]
This looks good I just had a logistical question 👍

File: node/packages/aws-lambda-sdk/instrument/lib/auto-send-spans.js (hunk 1 above)
Comment (medikoo) [reply node/packages/aws-lambda-sdk/instrument/lib/auto-send-spans.js:24]: Definitely, in the on going long span, there can be multiple captured events reported. And when they happen, there will be no new spans to report

Comment (Danwakeem) [reply node/packages/aws-lambda-sdk/instrument/lib/auto-send-spans.js:24]: @medikoo Ahh okay got it so these would be captured events for a parent span. Maybe it is the long span or some other one 👍 Thanks!

**Pull request merged**